sensors/test_deps.go

```diff
@@ -198,6 +239,7 @@ func getIMUWithErroringFunctions() *inject.MovementSensor {
 	imu.AngularVelocityFunc = func(ctx context.Context, extra map[string]interface{}) (spatialmath.AngularVelocity, error) {
 		return spatialmath.AngularVelocity{}, errors.New("invalid sensor")
 	}
+
```

Comment:
maybe remove this or add a new line below 
```
imu.LinearAccelerationFunc = func(ctx context.Context, extra map[string]interface{}) (r3.Vector, error) {
		return r3.Vector{}, errors.New("invalid sensor")
	}
```

Reply:
Removed all new lines to match patterns established in lidar functions